injector/LogInjector.py

```
@@ -0,0 +1,158 @@
+import ast
+import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)